tests/Finals.tex

```diff
@@ -1,85 +1,77 @@
+\textbf{SSPL:} The robot naturally interacts with a non-expert user. (TODO: new theme for this year) \\
 
-As it can be seen, the objectives are open enough that a story can be told around them which can include additional objectives that the team wants their robot to also solve. Thus, the teams are welcome to include in their demonstration any additional tasks to be solved, which can serve as a type of forum where they can present their own research. The innovation and success of these tasks will also be used as part of the score (as it is described later). In this regard, it is expected that teams present the scientific and technical contributions they submitted in both \iterm{team description paper} and the \iterm{RoboCup\char64Home Wiki}.
+\noindent Teams are free to use anything from the \AtHome{} scenario (see~\ref{sec:rules:scenario}) and to chose participants from other teams, the jury, or the audience. The themes are open to allow teams to create a story around them which incorporates additional tasks and abilities the team wants to present. Teams may provide a printed document to the jury (max 1 page) that summarizes the demonstrated robot capabilities and contributions. However, teams are discouraged to provide any material that would distract from their demonstration. It is recommended to spend only a short amount of time to explain the demonstration and rather let it speak for itself.
 
-In addition, teams may provide a printed document to the jury (max 1 page) that summarizes the demonstrated robot capabilities and contributions. However, teams are discouraged to provide any material that would distract from their demonstration.
 
-Story-telling is an important factor, so it is recommended to spend the least amount of time using the microphone to explain the demonstration and let the demonstration speak for itself.
-
-
-\section{Evaluating Juries for Final Demonstrations}
-The \iterm{Finals} are evaluated by two juries, here described.
+\section{Juries and Evaluation Criteria}
+\label{sec:finals:juries}
+The \iterm{Finals} are evaluated by two juries:
 
 \begin{enumerate}
-\item\textbf{League-internal jury:} The league-internal jury is formed by the Executive Committee. The evaluation of the league-internal jury is based on the following criteria:
+\item\textbf{League-Internal Jury:} Formed by the \EC{}. The evaluation is based on the following criteria:
   \begin{compactenum}
-  \item Efficacy/elegance of the solution
-  \item Innovation/contribution to the league of the additional tasks solved
-  \item Difficulty of the overall demonstration
+  \item Efficacy/elegance of the solution.
+  \item Innovation/contribution to the league.
```

Comment:
Contribution as in "new standard" for upcoming competitions or like scientific contribution as we discussed to raise output for scientific research?

Reply:
Both